src/query/expression/src/values.rs

```diff
@@ -1972,12 +1972,19 @@ impl ColumnBuilder {
             }
             ScalarRef::Map(col) => ColumnBuilder::Map(Box::new(ArrayColumnBuilder::repeat(col, n))),
             ScalarRef::Bitmap(b) => {
-                let rb =
-                    RoaringTreemap::deserialize_from(*b).expect("failed to deserialize bitmap");
-                let mut buf = vec![];
-                rb.serialize_into(&mut buf)
-                    .expect("failed to serialize bitmap");
-                ColumnBuilder::Bitmap(StringColumnBuilder::repeat(&buf, n))
+                if b.is_empty() {
+                    let rb = RoaringTreemap::new();
+                    let mut buf = vec![];
+                    rb.serialize_into(&mut buf).unwrap();
+                    ColumnBuilder::Bitmap(StringColumnBuilder::repeat(&buf, n))
```

Comment:
Using empty `vec![]` as the default value for `Bitmap` will cause a lot of problems, every time we deserialize it needs to check if the value is empty. So after some discussion, we think we can change the Bitmap's default value to an empty `RoaringTreemap`, which is the way you implemented it in the beginning. Sorry for the extra code changes.
This function just copies the value, there is no need to deserialize, we can directly call the `StringColumnBuilder::repeat` functions like `String` and `Variant` types.

Reply:
fine